src/pdfkit.cs

```diff
@@ -2118,4 +2180,22 @@ interface PdfViewDelegate {
 		UIViewController ParentViewController { get; }
 	}
 
+	[Mac (13,0), iOS (16,0), MacCatalyst (16,0)]
+	interface IPdfPageOverlayViewProvider {}
+
+	[Mac (13,0), iOS (16,0), MacCatalyst (16,0)]
+	[Protocol (Name = "PDFPageOverlayViewProvider")]
+	interface PdfPageOverlayViewProvider
+	{
+		[Abstract]
+		[Export ("pdfView:overlayViewForPage:")]
+		[return: NullAllowed]
+		UIView RequestPdfView (PdfView view, PdfPage page);
```

Comment:
I'm not sure about the naming here, `RequestPdfView` doesn't sound right (the method isn't returning a `PdfView`). Maybe something like this:

```suggestion
		UIView GetOverlayView (PdfView view, PdfPage page);
```

Reply:
Sounds good! I had a hard time naming this one so thanks haha